service/matching/version_sets.go

```diff
@@ -127,6 +131,13 @@ func UpdateVersionSets(clock hlc.Clock, data *persistence.VersioningData, req *w
 	return data, nil
 }
 
+func hashBuildID(buildID string) string {
+	bytes := []byte(buildID)
+	summed := sha256.Sum256(bytes)
+	// We don't need the padding
+	return strings.ReplaceAll(base64.URLEncoding.EncodeToString(summed[:]), "=", "")
```

Comment:
first, `RawURLEncoding` doesn't include padding.
second, just truncate instead:
```suggestion
	return base64.RawURLEncoding.EncodeToString(summed[:])[:20]
```
(20 base64 chars is 120 bits of entropy which is plenty)

Reply:
Sure